recipe-server/normandy/recipes/tests/test_models.py

```diff
@@ -283,6 +283,12 @@ def test_recipe_update_locales(self):
         recipe.update(locales=[])
         assert recipe.locales.count() == 0
 
+    def test_recipe_update_arguments(self):
+        recipe = RecipeFactory(arguments_json='')
+        recipe.update(arguments={'something': 'value'})
+        assert recipe.arguments == {'something': 'value'}
+        assert recipe.arguments_json == '{"something": "value"}'
```

Comment:
This second assertion seems redundant to me unless the code that turns the arguments into JSON is complicated...